### ISSUE TITLE
Decreased accuracy

### ISSUE THREAD

**Issue by blueskywwc** [burhanmudassar/pytorch-quant#1]
Hello, the accuracy rate after quantization has dropped by about 3%, and it has dropped too much. What is going on, thank you!

**Comment by burhanmudassar** [comment]
Hi!

For static quantization it's normal to have almost 5% drop. With quantization-aware training you should see the accuracy drop around 1 to 2% of the floating point model accuracy.

**Comment by blueskywwc** [comment]
Thank you for your reply, but for resnet18, resnet50, I quantify after training, the accuracy rate has only dropped by less than 0.3%, the difference between resnet and mobilenet is very big, what is the reason, thank you!

**Comment by burhanmudassar** [comment]
Hi, The why is hard to answer but I would guess that resnet are quite over-parameterized and have room to recover even with quantization. Mobilenet on the other hand is already quite small and doesn't have many parameters to recover from the quantization drop. You can read this reference where they talk about how its easy to prune larger networks than train a smaller network to that accuracy. The same arguments should apply to quantization as well

https://arxiv.org/pdf/1803.03635.pdf
Read footnote 1 on the first page

**Comment by blueskywwc** [comment]
Yes, the explanation I saw elsewhere is that the neural network is over-parameterized and contains enough redundant information, and cutting this redundant information will not cause a significant decrease in accuracy. The relevant evidence is that for a given quantization method, the accuracy gap between the FP32 network and the INT8 network is smaller for large networks, because the large networks are more parameterized.
thank you.
